example_scripts/example_runScripts.py

```diff
@@ -43,9 +44,25 @@
 from fitting_benchmarking import do_fitting_benchmark as fitBenchmarking
 from results_output import save_results_tables as printTables
 
-
 # SPECIFY THE SOFTWARE/PACKAGE CONTAINING THE MINIMIZERS YOU WANT TO BENCHMARK
-software = 'mantid'
+software = 'scipy'
+software_options = {'software': software}
+
+if len(sys.argv) > 1:
+    software_options['minimizer_file'] = current_path + sys.argv[1]
```

Comment:
Looks a lot better overall, but still have an issue with software_options have three attributes. If we keep this, these three options needs to be transparent in all methods they are used and documented